### PR TITLE
Sc 10616 GDS User UI : implement Identity Certificate Details feature

### PR DESCRIPTION
**Pull request by elysee15**
### Scope of changes
Sc 10616 GDS User UI : implement Identity Certificate Details feature

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria
https://app.shortcut.com/rotational/story/10616/gds-user-ui-implement-identity-certificate-details-feature

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  I have updated the dependencies list
- [ ]  I have recompiled and included new protocol buffers to reflect changes I made
- [ ]  I have added new test fixtures as needed to support added tests
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.

### CODE REVIEW

**Review by masskoder — APPROVED** [review]

File: web/gds-user-ui/src/components/CertificateDetails/CertificateDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+  HStack,
+  Menu,
+  MenuButton,
+  MenuItem,
+  MenuList,
+  Stack,
+  Text,
+  VStack
+} from '@chakra-ui/react';
+import { t, Trans } from '@lingui/macro';
```
Comment (masskoder) [inline web/gds-user-ui/src/components/CertificateDetails/CertificateDetails.tsx:16]: i think `trans` is coming from @lingui/react packages.

File: web/gds-user-ui/src/components/CertificateDetails/CertificateDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+      <Stack spacing={5}>
+        <HStack
+          border="1px solid #DFE0EB"
+          bg="#D8EAF6"
+          py={3}
+          px={8}
+          borderRadius="10px"
+          justifyContent="space-between">
+          <Text fontWeight={700} fontSize="lg">
+            <Trans>MAINNET Identity Certificate</Trans>
```
Comment (masskoder) [inline web/gds-user-ui/src/components/CertificateDetails/CertificateDetails.tsx:92]: i think its missing id in all trans tag

File: web/gds-user-ui/src/components/CertificateManagement/MainnetTestnetCertificates.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+                  <Trans>Actions</Trans>
+                </Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              {data && data.length ? (
+                data?.map((certificate, idx) => (
+                  <Tr key={idx} data-testid="table-row">
+                    <Td>{certificate.serial_number}</Td>
+                    <Td>{dayjs(certificate.issued_at).format(DATE_FORMAT)}</Td>
```
Comment (masskoder) [inline web/gds-user-ui/src/components/CertificateManagement/MainnetTestnetCertificates.tsx:83]: it's possible to create an abstraction of this date format? an external function like you did for `formatDisplayedValue`.

**Review by masskoder — APPROVED** [review]

File: web/gds-user-ui/src/components/CertificateDetails/CertificateDetails.tsx (hunk 2 above)
Comment (elysee15) [reply web/gds-user-ui/src/components/CertificateDetails/CertificateDetails.tsx:92]: we don't need to add an `id`

**Pull request merged**